PR Víctor Jarreta

VJarreta opened the pull request:
Web deploy using Heroku and GitHub, web documentation using Swagger. description.md contains the process (in spanish), guides and links to the web and documentation.

fjlopez commented:
Your code cannot be merged yet to the main branch. The springfox version must be updated to 3.0.0 because the version of Spring in the main branch is now 2.3.3.  If you apply the proposed changes below correctly, you will receive the :gift:.

Your must remove:

```gradle
    compile group: 'org.springframework.plugin', name: 'spring-plugin-core', version: '1.2.0.RELEASE' 
    compile group: 'io.springfox', name: 'springfox-swagger2', version: '2.9.2'
    compile group: 'io.springfox', name: 'springfox-swagger-ui', version: '2.9.2'   
    compile('org.springframework.boot:spring-boot-starter-data-rest')
    compile group: 'io.springfox', name: 'springfox-data-rest', version: '2.9.2'
```

And use instead:
```gradle
    implementation "io.springfox:springfox-boot-starter:3.0.0"
```

In addition, the description of your improvement must be written in English.

VJarreta commented:
Done. I changed description.md language and I modified build.gradle. I also had to create a new file system.properties, so Heroku would compile correctly.
